Lib/test/test_gettext.py

```diff
@@ -931,6 +932,10 @@ def test__all__(self):
         support.check__all__(self, gettext,
                              not_exported={'c2py', 'ENOENT'})
 
+    @cpython_only
+    def test_lazy_import(self):
+        ensure_lazy_imports("gettext", {"re"})
```

Comment:
```suggestion
        ensure_lazy_imports("gettext", {"re", "warnings", "locale"})
```
I'm getting these by searching for "import" in the source file and collecting the imports inside functions